lib/main.dart

```diff
@@ -21,12 +20,19 @@ import 'package:survey_frontend/presentation/screens/login_screen.dart';
 import 'package:survey_frontend/presentation/screens/survey/survey_end_screen.dart';
 import 'package:survey_frontend/presentation/screens/survey/survey_start_screen.dart';
 import 'package:survey_frontend/presentation/screens/welcome_screen.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
 
 void main() async {
   await GetStorage.init();
   String startScreenPath =
       _getStartScreenPath(); 
   runApp(GetMaterialApp(
+    title: 'Localizations Sample App',
```

Comment:
What is this title?

Reply:
It apears in reasent opened apps. I'll change it to smth more suitable for our project